Casks/d/devtoys.rb

```diff
@@ -1,15 +1,25 @@
 cask "devtoys" do
-  version "0.0.11"
-  sha256 "84f6b354546b1731204deb48daed53caacab277b524bcdd8d5e47d53fc47b413"
+  version "2.0.0.3"
+  sha256 :no_check
```

Comment:
Once the binaries are live, this can be updated with the correct values.

```suggestion
  sha256 arm:   "1652673cc22fee65a1ddd4800aad357f5ea607d124543d8b3a0dc8229be3c1d6",
         intel: "633018324dd247ea5a210600642c7ce34bbf93beeca32d7b175e68c29a91014d"
```

Reply:
Thanks! I actually already know the checksum. I will add them back.